src/backend/distributed/sql/citus--12.1-1--12.2-1.sql

```diff
@@ -7,6 +7,7 @@
 #include "udfs/start_management_transaction/12.2-1.sql"
 #include "udfs/execute_command_on_remote_nodes_as_user/12.2-1.sql"
 #include "udfs/mark_object_distributed/12.2-1.sql"
+#include "udfs/citus_unmark_object_distributed/12.2-1.sql"
```

Comment:
we first need to drop the current version of this function:

```suggestion

DROP FUNCTION pg_catalog.citus_unmark_object_distributed(oid, oid, int);
#include "udfs/citus_unmark_object_distributed/12.2-1.sql"

```

Reply:
We shouldn't. We will have two version. First one without check existence flag and in this case checkobjectexistence is true. If we want to intentionally set to false, then we will call the version with checkobjectexistence with false flag set